torch/_inductor/utils.py

```diff
@@ -1142,7 +1143,7 @@ def is_big_gpu(index_or_device: Union[int, torch.device] = 0) -> bool:
             return False
         return True
 
-    min_sms = 68  # 3080
+    min_sms = config.xpu.big_gpu_threshold if gpu_type == "xpu" else 68  # 3080
```

Comment:
```suggestion
    min_sms = 16 if device.type == "xpu" else 68  # 3080
```

Let's not put this in a config.

Reply:
@jansel  thanks for your time, please review the resolved part.